package.json

```diff
@@ -31,6 +30,17 @@
       "path": "node_modules/cz-conventional-changelog"
     }
   },
+  "nyc": {
+    "extends": "@istanbuljs/nyc-config-typescript",
+    "all": true,
+    "include": [
+      "src/**.ts"
+    ],
+    "exclude": [
+      "src/.**"
+    ],
+    "check-coverage": false
```

Comment:
Prevents low test coverage from failing the build. Once more tests are migrated over we can configure the desired thresholds and re-enable this.